src/main.py

```diff
@@ -0,0 +1,78 @@
+# sudoku_solver.py
+# author: Daniel Jauergui
+# date: 3-31-2015
+
+from configuration.configuration import *
+from file_manager.file_manager import *
+from menu import Menu
+from game import *
+
+
+def main_menu():
+    menu = Menu('Sudoku Solver')
+    menu.clear_items()
+    menu.add_item((1, 'Configuration Game', config_menu))
+    menu.add_item((2, 'Start Game', game_menu))
+    menu.add_item((0, 'Exit', None))
+    menu.ask()
+
+
+def config_menu():
+    menu = Menu('Sudoku Solver - Config Section')
```

Comment:
config_menu does not seems accurate for the method function, probably a show or display config_menu will be better